app/src/main/java/jp/developer/retia/frozenword/ui/activity/HabbitsActivity.kt

```diff
@@ -0,0 +1,153 @@
+package jp.developer.retia.frozenword.ui.activity
+
+import android.content.Context
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/verno3632/FrozenWord/blob/5d531f90da2a5c007e00fca4d5f9537c4641fdd2/app/src/main/java/jp/developer/retia/frozenword/ui/activity/HabbitsActivity.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>